packages/components/src/input-control/input-base.tsx

```diff
@@ -65,7 +66,7 @@ export function InputBase(
 		size = 'default',
 		suffix,
 		...props
-	}: InputBaseProps,
+	}: InputBaseProps & FlexProps,
```

Comment:
I believe this is the correct placement of `FlexProps`. See https://github.com/WordPress/gutenberg/pull/40119/commits/a6826535f9ac7e70d2050309638995f561d65783

Reply:
I think that normally we have "merged" prop types in the `types.ts` file (similarly to how the code was before the changes in this PR).

Is the reason for this to avoid that `InputControlProps` inherits `FlexProps` from `InputBaseProps` ?

Reply:
> I think that normally we have "merged" prop types in the types.ts file (similarly to how the code was before the changes in this PR).

Ah ok, I'll keep the logic in the types.ts file.

> Is the reason for this to avoid that [`InputControlProps` inherits `FlexProps` from `InputBaseProps`](https://github.com/WordPress/gutenberg/blob/a2b6391dd2bdeab975631ee822e048f12c3574f7/packages/components/src/input-control/types.ts#L133) ?

Correct 👍 

I verified that any `FlexProps` passed to `InputControl` would have been passed through (via `...props`) to an `<input>`, where they would have no effect. So I think we're ok.